com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -214,7 +214,11 @@ void UpdateEnvironmentWithInput(UnityRLInputProto rlInput)
                 ChannelCredentials.Insecure);
 
             m_Client = new UnityToExternalProto.UnityToExternalProtoClient(channel);
+#if UNITY_EDITOR
+            var result = m_Client.Exchange(WrapMessage(unityOutput, 200), deadline: DateTime.Now.AddSeconds(10000));
```

Comment:
Where does the 10000 come from? Should it be a constant?

Reply:
it's a mistake... changed it back